default/main.py

```diff
@@ -54,29 +54,36 @@ async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
     with client.context():
         for user in models.Userdata.query():
             async with aiohttp.ClientSession() as httpSession:
+
                 async with httpSession.get(
                         'https://www.virustotal.com/api/v3/intelligence/search?query={query}'.format(query=user.vt_query),
                         headers={'x-apikey': user.apikey},
-                        ssl=ssl_context
-                    ) as resp:
-                    js = {'api_key': user.apikey}
-                    js.update(await resp.json())
-                    
-                    if resp.status != 200:
-                        logging.error(resp.text())
+                        ssl=ssl_context) as vt_resp:
+
+                    payload = {'api_key': user.apikey}
+                    payload.update(await vt_resp.json())
+
+                    if vt_resp.status != 200:
+                        logging.error('VT query failed with {}: {}'.format(vt_resp.status, vt_resp.text()))
```

Comment:
```suggestion
                        logging.error(f'VT query failed with {vt_resp.status}: {vt_resp.text()}')
```

Use f-strings: https://docs.python.org/3/whatsnew/3.6.html#pep-498-formatted-string-literals

Reply:
Wow I didn't know about that thing